pkg/v1/tkg/managementcomponents/management_component_install.go

```diff
@@ -100,7 +100,7 @@ func installManagementPackageRepository(pkgClient tkgpackageclient.TKGPackageCli
 	repositoryOptions.PollInterval = packagePollInterval
 	repositoryOptions.PollTimeout = packagePollTimeout
 
-	return pkgClient.UpdateRepositorySync(repositoryOptions, tkgpackagedatamodel.OperationTypeInstall)
+	return pkgClient.UpdateRepositorySync(repositoryOptions, tkgpackagedatamodel.OperationTypeUpdate)
```

Comment:
I'm just curious, why has the operationType been updated here?

Reply:
It was a bug that got discovered during testing. When updating the repository to properly update and return the correct error on failure this needs to be set to `OperationTypeUpdate` operation stucks in waiting state.